Build/Config/Cpp.props

```diff
@@ -38,6 +38,8 @@
       <Optimization Condition="'$(Configuration)' != 'Release'">Disabled</Optimization>
       <BasicRuntimeChecks Condition="'$(Configuration)' == 'Release'"></BasicRuntimeChecks>
 
+      <AdditionalIncludeDirectories>$(SourceDirectory)$(ProjectName)\Source;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
This means we don't have relative include paths all over the place when including headers in the source folder.

`#include "../ViewModel.h"` vs `#include "ViewModels/ViewModel.h"`